c4gt-demo/backend/common/llm.py

```
@@ -0,0 +1,59 @@
+                        "stream": False,
+                        "format": "json",
+                    },
+                )
+            response.raise_for_status()
+            raw_response = response.json().get("response", "{}")
+            break
+        except httpx.ReadTimeout:
+            print(f"LLM ReadTimeout on attempt {attempt + 1}/{max_retries}")
+            if attempt == max_retries - 1:
+                return None
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)